lib/manager/kustomize/extract.ts

```diff
@@ -65,8 +66,8 @@ export function extractImage(image: Image): PackageDependency | null {
   if (image?.name && image.newTag) {
     return {
       datasource: datasourceDocker.id,
-      depName: image.name,
-      lookupName: image.name,
+      depName: image.newName ?? image.name,
```

Comment:
Please revert this, as renovate will use lookupName for version lookup.

Reply:
This is not updating the version of the `image.name`? So you do want the `newName` in the PR title (if I got it right - not sure about the distinction between the two)?

Reply:
I think we should leave `depName=image.name` and only change lookupName. So we use depName for visibility as before and use lookupName for fetching the new versions.

Reply:
What would the user expect in the PR title? e.g. `Update name to v1.1.0` or `Update newName to v1.1.0` ?

Reply:
👍

Reply:
https://github.com/renovatebot/renovate/pull/7356/commits/4e0b51a27b90595fe5f14991f7d07e009a31742e

Reply:
> What would the user expect in the PR title? 

I think it does not matter for the use case I described in this PR description (delayed registry proxying), however I do have one use case which I'm building this for and basically we're overriding a completely brand new container which does not even track the old container version, so saying `Update name to v1.1.0` is not even accurate, because the real `name` may as well be a `v6.7.8` for all we know and the version represents the `newName`.

I'm not sure if that's a common use case though.

Anyways, I'm happy with whatever decision here - happy to keep the commit I pushed, just as happy to revert.

Reply:
Ok, so we better announce the newName if available.

This way I suggest to assign `newName ?? name` to `depName` and remove `lookupName`, cause renovate will use `depName`, if `lookupName` is undefined.

Reply:
Ok, reverted earlier commit and pushed https://github.com/renovatebot/renovate/pull/7356/commits/e327232ddec82a7bd59519c6b41821ea9e48d5db